### ISSUE TITLE
Training in a smaller dataset

### ISSUE THREAD

**Issue by jwc19890114** [jwang0306/mta-lstm-pytorch#1]
Hi, I wanna ask that if training in a smaller dataset of a specific area, how to adjust the learning rate.
And where could I find the Radam optimizer? How about this [version](https://github.com/jettify/pytorch-optimizer)

THX, and thx for your program

**Comment by jwang0306** [comment]
Hi @jwc19890114 ,
> if training in a smaller dataset of a specific area, how to adjust the learning rate.

Maybe you can start with a smaller one.

> And where could I find the Radam optimizer?

RAdam can be found [here](https://github.com/LiyuanLucasLiu/RAdam), in the `radam` folder. Just download the `radam.py` and put it under the same directory of our notebook.
